tests/compare_hilites.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+model_formatter="/home/natasha/MurfelClickHouse/build/programs/clickhouse-format";
+new_formatter="/home/natasha/copyClickHouse/build/programs/clickhouse-format";
+compare="/home/natasha/copyClickHouse/tests/a.out"
```

Comment:
what is a.out ?

Reply:
That's the compiled hilite-aware comparator, it's in the same folder. I just committed this script as is for now because I'll probably need to change it anyway.

Reply:
`a.out` is meaningless, let's use `hilite-aware-comparator` instead